packages/jest-util/src/index.js

```diff
@@ -38,10 +34,29 @@ const createDirectory = (path: string) => {
   }
 };
 
+const getPackageRoot = () => {
+  const cwd = process.cwd();
+
+  // Is the cwd somewhere within an npm package?
+  let root = cwd;
+  while (!fs.existsSync(path.join(root, 'package.json'))) {
+    if (root === '/' || root.match(/^[A-Z]:\\/)) {
+      root = cwd;
+      break;
+    }
+    root = path.resolve(root, '..');
+  }
+
+  return root;
+};
+
+exports.Console = require('./Console');
 exports.createDirectory = createDirectory;
 exports.escapeStrForRegex = escapeStrForRegex;
-exports.FakeTimers = FakeTimers;
-exports.formatFailureMessage = formatFailureMessage;
-exports.JasmineFormatter = JasmineFormatter;
-exports.installCommonGlobals = installCommonGlobals;
+exports.FakeTimers = require('./FakeTimers');
+exports.formatFailureMessage = require('./formatFailureMessage');
+exports.getPackageRoot = getPackageRoot;
+exports.installCommonGlobals = require('./installCommonGlobals');
+exports.JasmineFormatter = require('./JasmineFormatter');
+exports.NullConsole = require('./NullConsole');
```

Comment:
can we sort this list by capitalized and lowercase and by alphabet within each individual section?